src/Service/VendorService/EbookCentral/EbookCentralVendorService.php

```diff
@@ -46,9 +44,12 @@ class EbookCentralVendorService extends AbstractBaseVendorService
      * @param string $resourcesDir
      *   The application resource dir
      */
-    public function __construct(EventDispatcherInterface $eventDispatcher, EntityManagerInterface $entityManager,
-                                LoggerInterface $statsLogger, string $resourcesDir)
-    {
+    public function __construct(
+        EventDispatcherInterface $eventDispatcher,
+        EntityManagerInterface $entityManager,
+        LoggerInterface $statsLogger,
+        string $resourcesDir
```

Comment:
Parameters should be one line

Reply:
It's the f***ing `apply-coding-standards` command that put them on separate lines!